#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-recordings.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: temporary calls should not be saved in /tmp, but in the sounds dir. This way
## DP: we can rename them to the correct names in an atomic command

@DPATCH@
diff -urNad freepbx-2.1.3.dfsg/amp_conf/astetc/extensions.conf /tmp/dpep.c51xrf/freepbx-2.1.3.dfsg/amp_conf/astetc/extensions.conf
--- freepbx-2.1.3.dfsg/amp_conf/astetc/extensions.conf	2006-11-26 15:15:22.000000000 +0200
+++ /tmp/dpep.c51xrf/freepbx-2.1.3.dfsg/amp_conf/astetc/extensions.conf	2006-11-26 15:17:11.000000000 +0200
@@ -543,11 +543,11 @@
 [macro-systemrecording]
 exten => s,1,Goto(${ARG1},1)
 
-exten => dorecord,1,Record(/tmp/${CALLERID(number)}-ivrrecording:wav)
+exten => dorecord,1,Record(custom/ivr-tmp-${CALLERID(number)}-ivrrecording:wav)
 exten => dorecord,n,Wait(1)
 exten => dorecord,n,Goto(confmenu,1)
 
-exten => docheck,1,Playback(/tmp/${CALLERID(number)}-ivrrecording)
+exten => docheck,1,Playback(custom/ivr-tmp-${CALLERID(number)}-ivrrecording)
 exten => docheck,n,Wait(1)
 exten => docheck,n,Goto(confmenu,1)
 
